src/App.js

```diff
@@ -14,6 +15,7 @@ export default function App() {
         </p>
       </header> */}
       <Route exact path="/" component={ Login } />
+      <Route exact path="/telaDeJogo" component={ TelaDeJogo } />
```

Comment:
```suggestion
      <Route exact path="/tela-de-jogo" component={ TelaDeJogo } />
```
Boa noite, para url, o recomendado é utilizar o kebab-case, aqui está a sugestão